.github/workflows/container-image.yml

```diff
@@ -4,54 +4,106 @@ on:
     - main
     tags:
     - 'v*'
-
 name: build container image
-
 jobs:
-  build:
+  build-x86_64:
+    name: Build x86_64 binary
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - run: rustup target add x86_64-unknown-linux-musl
+      - name: Setup musl for x86_64
+        run: |
+          curl -O https://musl.cc/x86_64-linux-musl-cross.tgz
+          tar -xzf x86_64-linux-musl-cross.tgz -C $HOME
+          echo "$HOME/x86_64-linux-musl-cross/bin" >> $GITHUB_PATH
+      - name: Build policy-server
+        env:
+          CC: x86_64-linux-musl-gcc
+        run: |
+          cargo build --target=x86_64-unknown-linux-musl --release
+      - name: Upload policy-server
+        uses: actions/upload-artifact@v2
+        with:
+          name: policy-server-x86_64
+          path: target/x86_64-unknown-linux-musl/release/policy-server
```

Comment:
I guess this why you changed the `.Dockerignore` to allow the `target` directory. I would just copy this file (and the aarch64 one) into the root of the git checkout. Obviously using two different names (like `policy-server-x86_64`)